source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
+      :tabid: project-file
+
+      1. Open a project file.
+
+         To open a file in |vscode-short|, click on the file name in the
+         file explorer. 
+
+      #. Highlight the code that you want to open in a playground.
+
+      #. Right click to select :guilabel:`Export Code to Playground`.
```

Comment:
(nit -- [right-click](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#r))
Always hyphenate. There are a couple instances. I'll just mark this one.

Reply:
Done!